vita3k/gui/src/allocations_dialog.cpp

```diff
@@ -48,7 +48,7 @@ void draw_allocations_dialog(GuiState &gui, EmuEnvState &emuenv) {
             continue;
 
         if (ImGui::TreeNode(fmt::format("{}: {}", generation_num, generation_name).c_str())) {
-            ImGui::Text("Range %08llx - %08llx.", generation_num * KiB(4), (generation_num + page.size) * KiB(4));
```

Comment:
This is wrong, generation_num * KiB(4) is a size_t, so the quantifier should be llx (or zx), lx works on Linux but not on Windows.

Reply:
is this enough? 
```cpp
ImGui::Text("Range 0x%08zx - 0x%08zx.", generation_num * KiB(4), (generation_num + page.size) * KiB(4));
```

Reply:
Yes, this should work. Or static cast them back to uint32_t and use %08x, as you want